canonical-web-and-design/ubuntu.com
# K8sDocs - Update 1.18 components for 1.18ck2

## Done

Updates the components and versioned charm pages for 1.18ck2. Now including actions.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the components page for 1.18  http://0.0.0.0:8001/kubernetes/docs/1.18/components
